### DISCUSSION TITLE
Instant audio playback

### DISCUSSION THREAD

**Discussion by Turbinya** [katspaugh/wavesurfer.js#3545] [Q&A]
Hi. I've been using wavesurfer for my needs for a long time and I haven't updated it for a long time. The current version I'm using is 5.2.0.  But recently I decided to update the version to the latest and I noticed one feature: It seems that in version 5.2.0 the audio was played from the first bytes received but now in version 7.7.2 the audio starts playing only after the full download.
Please tell me if this is true and why it changed?

**Comment by katspaugh** [comment]
That behaviour is still possible with a custom media element like this:

```js
const audio = new Audio('https://...')
audio.autoplay = true

const wavesurfer = WaveSurfer.create({
  container: '...',
  media: audio,
})
```
